frontend/src/containers/LibraryPage/LibraryPage.js

```diff
@@ -2,11 +2,40 @@ import { Grid } from '@material-ui/core';
 import React from 'react';
 import { UploadIcon } from '../../assets';
 import TopBar from '../../components/TopBar/TopBar';
+import UploadModal from '../../components/UploadModal/UploadModal';
 
 class LibraryPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showUploadModal: false
+    };
+  }
+
+  toggleUploadModal = () => {
```

Comment:
Maybe we should move this one to the TopBar. I know that it's a dummy component, but it would mean that we would not need to impleement the same function on every page that has an upload icon